.github/workflows/release.yml

```
@@ -52,6 +52,7 @@ jobs:
         id: cpr
         uses: peter-evans/create-pull-request@v3
         with:
+          base: main
```

```suggestion
          base: ${{ github.event.push.repository.default_branch }}
```

でいけると、もしこれから先ブランチ名を変えた時に気にしなくていいんですが……
（fork先でまたこの値が微妙に変わったりするのかなあという不安も少し抱えつつ）

---

> 先ブランチ名を変えた時に気にしなくていい

ですね！これができたらいいなと思ってました！
このactionはupstream/originそれぞれで使えれば良さそうな感じがします:+1: (originで使うのはあまりなさそうですが)